src/SonarScanner.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -141,9 +141,11 @@
   </Target>
 
   <Target Name="FailIfLowerThanMSBuild14"
-          Condition=" $(SonarQubeTempPath) != '' AND $(MSBuildToolsVersion) != '14.0' AND $(MSBuildToolsVersion) != '15.0'"
+          Condition=" $(SonarQubeTempPath) != '' AND $(MSBuildToolsVersion) != '14.0' AND $(MSBuildToolsVersion) != '15.0' AND $(MSBuildToolsVersion) != 'Current'"
           BeforeTargets="Build">
-    <Error Text="SonarQube analysis is only supported with MSBuild 14 or MSBuild 15" />
+    <Message Importance="high" Text="Current MSBuildToolsVersion: $(MSBuildToolsVersion)" />
+    <Message Importance="high" Text="Current MSBuildAssemblyVersion: $(MSBuildAssemblyVersion)" />
```

Comment:
Additional info to help debugging. I've set the importance to _high_ since this target is only executed when the error condition is met - that way, the messages will always appear in the log whenever the error does without the user having to re-run with a higher level of verbosity.